uwblueprint/don-efficace
# stripe webhooks with string cause ids

## Notion ticket link

[create Stripe webhooks for successful transactions](https://www.notion.so/uwblueprintexecs/create-Stripe-webhooks-for-successful-transactions-b04a9056da104634977f4e0420637ad1)



## Implementation description
* added webhook for successful transactions :)



## Steps to test
1. login to Stripe dashboard
2. set .env variables for stripe API keys (message Rosanne or Ben)
3. `yarn install` (if needed)
4. try running `stripe trigger checkout.session.completed`, and you can see the events successfully hit on the Stripe dashboard



## What should reviewers focus on?
* did we do this right :)


## Checklist
